src/assert.js

```diff
@@ -84,7 +84,7 @@ QUnit.assert = Assert.prototype = {
 	notOk: function( result, message ) {
 		message = message || ( !result ? "okay" : "failed, expected argument to be falsy, was: " +
 			QUnit.dump.parse( result ) );
-		this.push( !result, result, false, message, true );
+		this.push( !result, result, false, message );
```

Comment:
It's weird how `notOk` is a different kind of a negative assertion. It would be negative if the expected value was true, but we are actually checking if "*actual is equivalent to false*", and not "*actual is __not__ equivalent to false*".

This fix ends up reasonable.